src/i18n/it/ui.ts

```diff
@@ -0,0 +1,119 @@
+export default {
+	'a11y.skipLink': 'Vai al Contenuto',
+	'a11y.sectionLink': 'Sezione intitolata',
+	'navbar.a11yTitle': 'Cima',
+	// Site settings
+	'site.title': 'Documentazione di Astro',
+	'site.description': 'Costruisci siti web veloci con meno JavaScript sul client.',
+	'site.og.imageSrc': '/default-og-image.png?v=1',
+	'site.og.imageAlt':
+		'Logo di Astro su una distesa di stelle, con un pianeta viola simile a Saturno che fluttua in primo piano a destra',
+	// Left Sidebar
+	'leftSidebar.a11yTitle': 'Principale',
+	'leftSidebar.learnTab': 'Impara',
+	'leftSidebar.referenceTab': 'Riferimento',
+	'leftSidebar.viewInEnglish': 'Visualizza in Inglese',
```

Comment:
Should this be in actual English?

Reply:
I really don't like "Riferimento"... I'd rather leave it in English, but the rest are very well translatable. I saw that in the French version these are translated as well.

The only doubt I have is about "Impara". The word "Learn" is not that new in Italy and we may be able to let it slip through.

Reply:
I left "Impara" untouched for now. If it ends up being a better choice I'll change it when needed.